python/mxnet/gluon/block.py

```diff
@@ -1040,7 +1052,12 @@ def register_child(self, block, name=None):
         super(HybridBlock, self).register_child(block, name)
         self._clear_cached_op()
 
-    def hybridize(self, active=True, **kwargs):
+    def hybridize(self, active=True, backend=None, backend_args=None, **kwargs):
+        self._backend = backend
+        if backend_args is None:
+            self._backend_args = {}
+        else:
+            self._backend_args = backend_args
```

Comment:
we need to enforce users to pass a dictionary (since user may pass a string), so we need to add a check below before assign it to _backend_args

Reply:
lets add something like
```
if isinstance(backend_args, dict)
```

Reply:
Fixed. Thanks.

Reply:
we still need an `else` block.
```
else:
    self._backend_args = {}
```

Reply:
Okay, we don't need `else` as it is initialized to {}.